tests/admin_views/tests.py

```diff
@@ -1704,6 +1753,18 @@ def test_change_view(self):
             response = self.client.post(change_url_2, {'name': 'changed'})
             self.assertEqual(RowLevelChangePermissionModel.objects.get(id=2).name, 'changed')
             self.assertRedirects(response, self.index_url)
+            response = self.client.get(change_url_3)
+            self.assertEqual(response.status_code, 200)
+            response = self.client.post(change_url_3, {'name': 'changed'})
+            self.assertEqual(response.status_code, 302)
```

Comment:
Is it normal that the error code is 302 redirect ? I was expecting a 403 as well.